crates/bevy_animation/src/animatable.rs

```diff
@@ -1,3 +1,5 @@
+//! Traits and types for allowing things to be interpolated between two values.
```

Comment:
I had to add this doc-comment, to stop clippy from yelling and allow CI to work.

However, I feel this doc-comment could be improved. So any suggestions you might have would be appreciated!